content/blog/2023/07/16/2023-07-16-unreleased-production-changes-probe.adoc

```diff
@@ -79,5 +79,5 @@ Starting from ground zero is tough, whether implementing a new Java library or s
 However, with correct mentorship and guidance, you can rise up faster.
 
 == Links
-For more information or to find answers to any questions you might have, head to the official GSoC 2023 project link:/projects/gsoc/2023/projects/add-probes-to-plugin-health-score/[Adding Probes to  "Plugin Health Score"] description page.
+For more information or to find answers to any questions you might have, head to the official GSoC 2023 project link:../../../../projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc[Adding Probes to  "Plugin Health Score"] description page.
```

Comment:
```suggestion
For more information or to find answers to any questions you might have, head to the official GSoC 2023 project link:/projects/gsoc/2023/projects/add-probes-to-plugin-health-score/[Adding Probes to  "Plugin Health Score"] description page.
```
Same suggestion as the other link, it appears to work as originally written so it's not clear why the change was made.